apps/api/.env.example

```diff
@@ -1,24 +1,24 @@
 # Please modify the following environment variables accordingly.
 
 DATABASE_URL=mysql://ctims:ctims@localhost:3306/ctims
-KEYCLOAK_URL=http://localhost:8080
+KEYCLOAK_URL=http://localhost:8181
 KEYCLOAK_CLIENT_ID=ctims
 
 # Keycloak client uuid. This is different from the id above.
 # If you cannot find it in the client settings, the URL usually includes it. Example: http://localhost:8080/admin/master/console/#/ctims/clients/3813811a-***/settings, 3813811a-*** is the uuid
-KEYCLOAK_CLIENT_UUID=<uuid>
+KEYCLOAK_CLIENT_UUID=15d6ff8e-a439-45c9-8644-615d6f414309
 
 # We suggest creating a new realm instead of using the default master realm.
 KEYCLOAK_REALM=ctims
 
 # This is the client secret that stored under the Credentials tab in the client page.
 # The realm needs to have client authentication enabled to see the Credentials tab.
-KEYCLOAK_CLIENT_SECRET=<secret>
+KEYCLOAK_CLIENT_SECRET=GD0UkeywCNobyDdqmwOiPrHlkMB1uLzO
 
 # A client with service account enabled.
 # It does not need to be a separate client. If your above client already has service account enabled, use the same ID here.
-KEYCLOAK_ADMIN_CLIENT_ID=ctims-admin
-KEYCLOAK_ADMIN_CLIENT_SECRET=<secret>
+KEYCLOAK_ADMIN_CLIENT_ID=ctims
+KEYCLOAK_ADMIN_CLIENT_SECRET=GD0UkeywCNobyDdqmwOiPrHlkMB1uLzO
```

Comment:
You will have your own .env.example file, this file should not be modified unless changes are applicable to all local installations.

Reply:
reverted back to original form